lib/src/io.dart

```diff
@@ -575,13 +575,21 @@ Future<bool> confirm(String message) {
 }
 
 /// Writes [prompt] and reads a line from stdin.
-Future<String> stdinPrompt(String prompt) {
+Future<String> stdinPrompt(String prompt, {bool echoMode}) {
```

Comment:
Maybe we should move these utilities into:
https://github.com/dart-lang/cli_util

And make sure that it's possible to do more than one prompt in a process.

This is not important, just a possible cleanup we could do, which would also allow other Dart programs to benefit from the work we're doing.